product_docs/docs/epas/14/epas_compat_cat_views/04_all_constraints.mdx

```diff
@@ -15,15 +15,15 @@ The `ALL_CONSTRAINTS` view provides information about the constraints placed on
 | `schema_name`       | `TEXT`    | Name of the schema in which the constraint belongs.                                                                                                                                                                                                            |
-| `constraint_name`   | `TEXT`    | The name of the constraint.                                                                                                                                                                                                                                    |
+| `constraint_name`   | `TEXT`    | Name of the constraint.                                                                                                                                                                                                                                    |
 | `constraint_type`   | `TEXT`    | The constraint type. Possible values are: `C` – check constraint; `F` – foreign key constraint; `P` – primary key constraint; `U` – unique key constraint; `R` – referential integrity constraint; `V` – constraint on a view; `O` – with read-only, on a view |
 | `table_name`        | `TEXT`    | Name of the table to which the constraint belongs.                                                                                                                                                                                                             |
 | `search_condition`  | `TEXT`    | Search condition that applies to a check constraint.                                                                                                                                                                                                           |
 | `r_owner`           | `TEXT`    | Owner of a table referenced by a referential constraint.                                                                                                                                                                                                       |
 | `r_constraint_name` | `TEXT`    | Name of the constraint definition for a referenced table.                                                                                                                                                                                                      |
-| `delete_rule`       | `TEXT`    | The delete rule for a referential constraint. Possible values are: `C` – cascade; `R` – restrict; `N` – no action                                                                                                                                              |
+| `delete_rule`       | `TEXT`    | Eelete rule for a referential constraint. Possible values are: `C` – cascade; `R` – restrict; `N` – no action                                                                                                                                              |
```

Comment:
I think it is typo. It should be `Delete` not `Elete`

Reply:
Fixed